src/test/java/com/lafaspot/icap/client/IcapClientTest.java

```diff
@@ -27,6 +27,7 @@
 import org.testng.annotations.BeforeClass;
```

Comment:
Renamed class to IcapClientIT. My tests can't be copied to IcapClientTest because it expects session to be created to test 0 byte size files.